raiden-dapp/vue.config.js

```diff
@@ -1,10 +1,19 @@
+const fs = require('fs');
 const path = require('path');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
+const VersionFile = require('webpack-version-file-plugin');
+const { DefinePlugin } = require('webpack'); // eslint-disable-line import/no-extraneous-dependencies
 const { InjectManifest } = require('workbox-webpack-plugin');
 
 const sourceDirectoryPath = path.resolve(__dirname, 'src');
 const distributionDirectoryPath = path.resolve(__dirname, 'dist');
 
+function getPackageVersion() {
+  const packageFilePath = path.resolve(__dirname, 'package.json');
```

Comment:
Maybe import the JSON instead of reading the file explicitly?
```js
var pjson = require('./package.json');
console.log(pjson.version);

Reply:
:heavy_check_mark: